presto-spark-base/src/main/java/com/facebook/presto/spark/execution/http/PrestoSparkHttpWorkerClient.java

```diff
@@ -199,8 +220,79 @@ public ListenableFuture<BaseResponse<TaskInfo>> updateTask(
                 createAdaptingJsonResponseHandler(taskInfoCodec));
     }
 
+    public ListenableFuture<BaseResponse<ServerInfo>> getServerInfo()
```

Comment:
This method does not seem to be used publicly, or more than one place. Shall we remove this method, or change it to private?